README.md

```diff
@@ -26,6 +26,33 @@ To run tests, run `rake db:test:prepare; rspec`.
 
 You'll need to configure cloudinary, [including adding a cloudinary.yml file](http://cloudinary.com/documentation/rails_integration).
 
+#### Running Frolfr in Docker
+If process isolation is your bad then you're in luck - the Frolfr API is full containerized! Make sure you have docker fully setup and running in your local environment and then do the following to get frolfr running:
+```
+# build images
+docker-compose build
+
+# start the containers
+docker-compose up
+```
+
+You can now access Frolfr at 0.0.0.0!
```

Comment:
Can't you already access `frolfr-server` if you just type `unicorn` or `rails s`? What does "access" mean?